### ISSUE TITLE
Image manipulation broken/limited?

### ISSUE THREAD

**Issue by Tekl** [typemill/typemill#210]
Does someone has a clue, why this gives me the normal uncropped images from the live folder:

``twig
{{ assets.image(post.meta.heroimage).resize(100,100).src() }}
``

and this the expected cropped images from the custom folder?

``twig
{{ assets.image(post.meta.heroimage).resize(460,460).src() }}
``

The custom folder is writeable as using values like 400 also work. 200 or 250 does not work.

**Comment by trendschau** [comment]
I cannot reproduce that behavior, for me it works fine with all sizes. From code logic it only returns the original image url if the reiszed image cannot be stored.

I changed the validation from ctype_digit (expects string) to is_int (expects integer) for next release. Maybe that also produced an error and it works with the next reiease.

If not, I need more details or the exact code with test-image to reproduce this case.

**Comment by Tekl** [comment]
I tried other values. 50 works well, 80 not.

This is the code:

```twig
{% elseif item.contains == 'posts' %}

			<ol class="postlist" reversed>

			{% for element in item.folderContent %}

				{% set post = getPageMeta(settings, element) %}
				{% set date = element.order[0:4] ~ '-' ~ element.order[4:2] ~ '-' ~ element.order[6:2] %}

				<li class="post-entry" role="article">
                    {% if post.meta.heroimage %}
                        {% if item.slug == "news" %}                                                
                            <div class="border postimage"><a href="{{ element.urlAbs }}"><img src="{{ assets.image(post.meta.heroimage).resize(100,100).src() }}" alt="{{ post.meta.heroimagealt }}" width="50" /></a></div>
                        {% else %}
                            <div class="border postimage"><a href="{{ element.urlAbs }}"><img src="{{ assets.image(post.meta.heroimage).resize(460,460).src() }}" alt="{{ post.meta.heroimagealt }}" width="230" /></a></div>
                        {% endif %}
                    {% endif %}
					<header>
                        <h3><a href="{{ element.urlAbs }}">{{ post.meta.title }}</a></h3>
                        <small><time datetime="{{date}}">{{ date | date("d.m.Y") }}</time> von {{ post.meta.author }}</small>
                    </header>
                    {% if item.slug == 'blog' %}
                        <p>
                            {{ post.meta.description }}
                        </p>
                        <p class="readmore"><a href="{{ element.urlAbs }}">Weiterlesen</a></p>
                    {% endif %} 
				</li>
								
			{% endfor %}
			
			</ol>

		{% endif %}
```

**Comment by trendschau** [comment]
I tested it with version 1.3.8 again and for me it works. 

Please note that (due to a wrong validation rule) the function expects a string, so right now it works with `{{ assets.image(post.meta.heroimage).resize('100','100').src() }}`. This will change with the next release so I suggest that you wait for version 1.4.0

**Comment by trendschau** [comment]
published with version 1.4.0 today
